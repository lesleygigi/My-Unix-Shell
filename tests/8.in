/usr/bin/seq 2 2 8 > /tmp/test.out
cat /tmp/test.out
