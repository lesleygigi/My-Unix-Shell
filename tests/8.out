/usr/bin/seq 2 2 8 > /tmp/test.out
cat /tmp/test.out
2
4
6
8
